crates/proto/src/serialize/mod.rs

```diff
@@ -17,3 +17,4 @@
 //! Contains serialization libraries for `binary` and text, `txt`.
 
 pub mod binary;
+pub mod txt;
```

Comment:
I think we should make txt serialization a default off feature. It wold reduce the code size for the resolver use case. What do you think?

Reply:
Makes sense. The difference in clean `--no-default-features` debug builds for -proto is about 5% (0.3s) on my machine, but that includes a bunch of dependencies so it's probably still significant.